component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/auth/DefaultApi.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.am.analytics.publisher.auth;
+
+import feign.RequestLine;
+
+/**
+ * Analytics Auth API
+ *
+ * <p>This is a RESTFul API for authenication information to communcate with event hub 
+ *
+ */
+public interface DefaultApi  {
```

Comment:
Why do we call it DefaultAPI?

Reply:
I use a code generator and let use it. This is how it derived the name. But I can rename it.